docs/configure.rst

```diff
@@ -0,0 +1,113 @@
+.. highlight:: console
+.. _configure:
+
+
+=========
+Configure
+=========
+
+
+Install Croud CLI
+=================
+
+This page shows you how to download and install the `Croud CLI`_. Croud is the dedicated command-line interface (CLI) tool to interact with CrateDB Cloud.
+The CLI provides specific commands to create and manage CrateDB Cloud
+organizations, projects, and resources. You can also use it to deploy CrateDB products such as clusters.
+
+In this tutorial, we refer to the *Croud CLI* as *Croud*.
+
+.. NOTE::
+
+   Croud requires `Python 3.6 or higher`_.
+
+.. SEEALSO::
+
+   Check out the `Croud CLI Reference`_ for an outline of all available Croud
+   commands, command-line options, and usage instructions.
+
+To install Croud, start your system's command-line application and run the
+following command:
+
+.. code-block:: console
+
+    sh$ python3 -m pip install --user -U croud
+
+Next, verify if the installation was successful by running:
+
+.. code-block:: console
+
+    sh$ croud -v
+
+If Croud was successfully installed, the command outputs the Croud version
+number, like so:
+
+.. code-block:: console
+
+   croud 0.16.0.
+
+
+.. _sign-in:
+
+Sign in to CrateDB Cloud
+========================
+
+This sections shows you how to sign in to CrateDB Cloud using `Croud`_.
+
+.. NOTE::
+
+  This document assumes that you have `created a CrateDB Cloud account`_.
+
+Run the ``croud login`` command to sign in to CrateDB Cloud:
+
+.. code-block:: console
+
+    sh$ croud login
+
+The CrateDB Cloud sign-in page will open in a new browser tab.
+
+.. image:: _assets/img/cloud-sign-in-page.png
+
+Provide your username and password and select *Sign in*.
+
+In another new browser tab, you should get the message:
+`You have successfully logged into CrateDB Cloud!`
+
+Close the tab and return to Croud.
+
+Verify that you are signed in by giving the ``croud`` command and passing in
+``me`` to retrieve the data of the user that is currently signed in:
+
+.. code-block:: console
+
+    sh$ croud me
+
+If you're signed in, then CrateDB Cloud displays your email address and
+username, like so:
+
+.. code-block:: console
+
+   +--------------------+--------------+
+   | email              | username     |
+   |--------------------+--------------|
+   | youruser@email.com | yourusername |
+   +--------------------+--------------+
+
+In case you're not signed in, run the ``croud`` command and pass in ``login`` to
+sign in to CrateDB Cloud:
+
+.. code-block:: console
+
+  sh$ croud login
+
+
+Next steps
+==========
+
+You have now installed Croud and you are signed in to CrateDB Cloud. Next, let's
```

Comment:
"You have successfully installed Croud and subsequently signed into CrateDB Cloud. Now you can deploy your first CrateDB cluster."